### PR TITLE
Jsnapy release137 update

### PR DESCRIPTION
**Pull request by chidanandpujar**
### What does this PR do?
jsnapy release update to 1.3.8.dev0

### What issues does this PR fix or reference?
jsnapy release update to 1.3.8.dev0

### Tests written?

Yes/No

### CODE REVIEW

File: lib/jnpr/jsnapy/version.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 # Copyright (c) 1999-2018, Juniper Networks Inc.
 #
-# All rights reserved.
+# All rights reserved .
```
Comment (dineshbaburam91) [inline lib/jnpr/jsnapy/version.py:5]: Please revert back the string

Comment (dineshbaburam91) [inline lib/jnpr/jsnapy/version.py:5]: Kindly revert back to old string

**Review by dineshbaburam91 — COMMENTED** [review]
.

File: lib/jnpr/jsnapy/version.py (hunk 1 above)
Comment (chidanandpujar) [reply lib/jnpr/jsnapy/version.py]: Thanks , updated the string

**Pull request merged**